.jshintrc

```diff
@@ -1,107 +1,142 @@
+	"supernew"      : false,     // true: Tolerate `new function () { ... };` and `new Object;`
+	"validthis"     : false,     // true: Tolerate using this in a non-constructor function
 
-    // Environments
-    "browser"       : true,     // Web Browser (window, document, etc)
-    "browserify"    : false,    // Browserify (node.js code in the browser)
-    "couch"         : false,    // CouchDB
-    "devel"         : false,     // Development/debugging (alert, confirm, etc)
-    "dojo"          : false,    // Dojo Toolkit
-    "jasmine"       : false,    // Jasmine
-    "jquery"        : true,    // jQuery
-    "mocha"         : false,     // Mocha
-    "mootools"      : false,    // MooTools
-    "node"          : true,    // Node.js
-    "nonstandard"   : false,    // Widely adopted globals (escape, unescape, etc)
-    "prototypejs"   : false,    // Prototype and Scriptaculous
-    "qunit"         : false,    // QUnit
-    "rhino"         : false,    // Rhino
-    "shelljs"       : false,    // ShellJS
-    "worker"        : false,    // Web Workers
-    "wsh"           : false,    // Windows Scripting Host
-    "yui"           : false,    // Yahoo User Interface
+	// Environments
+	"browser"       : true,     // Web Browser (window, document, etc)
+	"browserify"    : false,    // Browserify (node.js code in the browser)
+	"couch"         : false,    // CouchDB
+	"devel"         : false,     // Development/debugging (alert, confirm, etc)
+	"dojo"          : false,    // Dojo Toolkit
+	"jasmine"       : false,    // Jasmine
+	"jquery"        : true,    // jQuery
+	"mocha"         : false,     // Mocha
+	"mootools"      : false,    // MooTools
+	"node"          : true,    // Node.js
+	"nonstandard"   : false,    // Widely adopted globals (escape, unescape, etc)
+	"prototypejs"   : false,    // Prototype and Scriptaculous
+	"qunit"         : false,    // QUnit
+	"rhino"         : false,    // Rhino
+	"shelljs"       : false,    // ShellJS
+	"worker"        : false,    // Web Workers
+	"wsh"           : false,    // Windows Scripting Host
+	"yui"           : false,    // Yahoo User Interface
 
-    // Custom Globals
-    "globals"       : {
-        "_"           : true,
-        "Assets"      : true,
-        "Blaze"       : true,
-        "crypto"      : true,
-        "EJSON"       : true,
-        "FlowRouter"  : true,
-        "Meteor"      : true,
-        "Npm"         : true,
-        "Package"     : true,
-        "Promise"     : true,      // Avoid "redefinition of Promise" warning
-        "Random"      : true,
-        "ReactiveVar" : true,
-        "RocketChat"  : true,
-        "s"           : true,
-        "Session"     : true,
-        "swal"        : true,
-        "t"           : true,
-        "TAPi18n"     : true,
-        "Template"    : true,
-        "TimeSync"    : true,
-        "toastr"      : true,
-        "Tracker"     : true
-    }
+	// Custom Globals
+	"globals"       : {
```

Comment:
I recommend that set to `false` if you expect that these variables to read-only. Because there is the following description .

>Setting it to `false` will trigger JSHint to consider that variable read-only.

See also: http://jshint.com/docs/options/#globals